src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -121,7 +121,7 @@ public IReadOnlyNetworkPeerCollection ConnectedPeers
 
             this.Parameters.Version = this.NodeSettings.ProtocolVersion;
 
-            nodeStats.RegisterStats(this.AddComponentStats, StatsType.Component);
+            nodeStats.RegisterStats(this.AddComponentStats, StatsType.Component, 1100);
```

Comment:
Whats this magic number ;)

Reply:
Priority.
All components register stats and are displayed sorted by this number. Bigger the number- more priority.